admin_manual/configuration_server/occ_command.rst

```diff
@@ -629,11 +629,22 @@ one must be specified.
 entries that have no matching entries in the storage table. 
 
 You may transfer all files and shares from one user to another. This is useful 
-before removing a user::
+before removing a user. 
+
+For example, to move all files from ``<source-user>`` to ``<destination-user>``, use the following command:
+
+::
 
  sudo -u www-data php occ files:transfer-ownership <source-user>
  <destination-user>
 
+You can also move a limited set of files from ``<source-user>`` to ``<destination-user>``, by making use of the ``--path`` switch, as in the example below. 
+In it, only files (and folders) under ``data/folders/to/move`` will be moved.
```

Comment:
The wording needs to be changed that the whole folder "data/$sourceUser/files/path/to/move" will be transferred to "data/$targetUser/files/transferred $timestamp/move", not only its contents.

Because the current wording is implicitly saying that only the contents will be transferred, which would be like doing "mv data/$sourceUser/path/to/move/* data/$targetUser/transferred $timestamp/"

Note I don't remember the exact format for the transferred folder, to be verified.